portal-web/test/functional/com/liferay/portalweb/MyDMTests/MyDMTests.testcase

```diff
@@ -0,0 +1,91 @@
+@component-name = "portal-document-management"
+definition {
+    property testray.main.component.name = "Document Management";
+
+    	setUp {
+		task ("Opening the portal and doing the Login") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Creating a Page with a Documents and Media widget") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Documents and Media page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = "1",
+				groupName = "Guest",
+				layoutName = "Documents and Media page",
+				widgetName = "Documents and Media");
+				}
+
+        task ("Entering The Documents and Media page") {
```

Comment:
Oi @Natanaelbenvindo, eu acho que você poderia colocar para:

Entering in the documents and media page